EduNLP/SIF/parser/parser.py

```diff
@@ -226,7 +226,16 @@ def get_token(self):
                     print("Warning: there is some chinese characters in formula!")
                     self.warnning = 1
                     flag = 0
-                self.head += 1
+                    self.head += 1
+                elif flag and ch_informula == '\n':
```

Comment:
So, whether the deletion happens in is_sif?

Reply:
The return value of is_sif  is True or False. Although the text is modified, is_sif will return False.

Reply:
Negative, is_sif should not modify the original text

Reply:
Got it, I'll delete this modification.

Reply:
Have you finished it?

Reply:
Yes, I have done it. Please check if there is any problem.